layout.html

```diff
@@ -8,5 +8,6 @@
 </head>
 <body>
   <main class="markdown-body"></main>
+  <script src="https://unpkg.com/marky-deep-links@1.0.1/marky-deep-links-standalone-min.js"></script>
```

Comment:
[`marky-deep-links`](https://github.com/Flet/marky-deep-links) is a little snippet of code that will ensure navigating to `#niceanchor` will actually navigate to `#user-content-niceanchor` since all anchors are prefixed to ensure [DOM Clobbering](https://github.com/Flet/markdown-it-github-headings#why-should-i-prefix-heading-ids) does not happen.

Reply:
I'm not sure I like adding this to the default layout. I think the `marky-deep-links` might make more sense as something the user can add on their own. Prefixing everything with `user-content-` might make sense for a large website with a lot of user generated content, like github or npm, but not necessarily for a small website generated from markdown with `sitedown`.

Reply:
ah, I understand the need for `marky-deep-links` now. this seems a bit like over-optimization though. my site definitely doesn't need every ID to be prefixed `user-content-`.

While I understand the concern about DOM clobbering, this is more of an issue if the user is using IDs for CSS (which is a bad idea), or for JS (which is generally not a great idea either), and the default use case of sitedown has no CSS and no JS. For that reason I'd rather not include this snippet and the prefixing out of the box by default.

I see the `prefixHeadingIds` is set to true by default here: https://github.com/Flet/markdown-it-github-headings#options-and-defaults -- is there a way to make this optional in sitedown as well?